Ecal/src/Ecal/CLUE.cxx

```diff
@@ -157,7 +157,7 @@ std::vector<std::shared_ptr<CLUE::Density>> CLUE::setup(
       if (y < 0) {
         j = -j;
         y = (j + 0.5) * dc_;
-      } else
+      } else                  // TODO i think this 1 should be a j
```

Comment:
I dont understand this TODO, but maybe it should be an issue?

Reply:
maybe. i haven't had time to investigate this. but i think by symmetry the lopp over j and the loop over i should work the same. the comment refers to the line below